static/app/views/issueDetails/streamline/solutionsSection.tsx

```diff
@@ -0,0 +1,81 @@
+import {useRef} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import useDrawer from 'sentry/components/globalDrawer';
+import {useGroupSummary} from 'sentry/components/group/groupSummary';
+import Placeholder from 'sentry/components/placeholder';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Event} from 'sentry/types/event';
+import type {Group} from 'sentry/types/group';
+import {IssueCategory} from 'sentry/types/group';
+import type {Project} from 'sentry/types/project';
+import {SidebarSectionTitle} from 'sentry/views/issueDetails/streamline/sidebar';
+import {SolutionsDrawer} from 'sentry/views/issueDetails/streamline/solutionsDrawer';
+
+const isSummaryEnabled = (hasGenAIConsent: boolean, groupCategory: IssueCategory) => {
+  return hasGenAIConsent && groupCategory === IssueCategory.ERROR;
+};
+
+export default function SolutionsSection({
+  group,
+  project,
+  event,
+}: {
+  group: Group;
+  project: Project;
+  event?: Event;
```

Comment:
trying to encourage people to be more strict on these
```suggestion
  event: Event | undefined;
```